packages/api-gateway/src/drivers/http/routes/geolocation/index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const { geolocationAdapters } = require("../../adapters");
+const { getAddressSchema } = require("./schema");
+
+async function geolocationRouter(fastify) {
+  await fastify.get(
+    "/address",
+    {
+      schema: {
+        querystring: getAddressSchema,
```

Comment:
Can you add `tags` and `description` to your schema?

As part of #118 I added `Swagger-ui` to the `api-gateway`, and swagger uses this properties to group and describe the endpoints.

You can see a full example here: https://github.com/Platzi-Master-C9/booking-services/tree/master/packages/api-gateway#add-a-new-endpoint

```suggestion
        description: 'Returns the address of the provided geolocation coordinates',
        tags: ['Geolocation'],
        querystring: getAddressSchema,
```

Reply:
Ok, I will add swagger. Thank you for the comment